test/index.test.js

```diff
@@ -28,6 +28,12 @@ describe('debugnyan', () => {
     expect(logger.level()).toEqual(Logger.FATAL + 1);
   });
 
+  it('should have the debug level set to info', () => {
```

Comment:
I'd go with something that more directly reflects the intention of the test:

```suggestion
  it('should allow customizing the logging level', () => {
```

WDYT?